### PR TITLE
Add super_admin attribute to BypassAuth::Login

### PR DESCRIPTION
**Pull request by texasjusticar**
https://bypassmobile.atlassian.net/browse/DEV-413

To make it easier to restrict certain actions to super_admin, expose the field in the BypassAuth::Login so we can fetch the boolean straight from the auth servers response.

There were also some broken tests, so I fixed them....I think

### CODE REVIEW

File: spec/bypass_auth/helpers_spec.rb [hunk 1]
```diff
@@ -94,7 +94,7 @@ def index
       stub_request(:get, /session/).to_return(body: fixture('session.json'))
       allow(controller).to receive(:get_token) {'sessiontoken'}
       session = controller.validate_session
-      expect(session).to be_an_instance_of(BypassAuth::Session)
+      expect(session).to be_an_instance_of(OpenStruct)
```
Comment (trevrosen) [inline spec/bypass_auth/helpers_spec.rb]: I'm not following why the expectation here is matching the doc string: `should return true if header is valid`...?

Comment (texasjusticar) [inline spec/bypass_auth/helpers_spec.rb]: The spec was originally written by our intern 2 summers ago :shrug:

Comment (texasjusticar) [inline spec/bypass_auth/helpers_spec.rb]: @trevrosen updated

**Comment by trevrosen** [comment]
:shipit:

**Comment by jspies** [comment]
:ship:

**Pull request merged**